classes/interactable/interactable_dialog.gd

```diff
@@ -25,8 +25,6 @@ func _interact_with(body):
 	body.vel = Vector2.ZERO
 	body.locked = true
 	body.sign_frames = 1
-	if back_sprite:
-		body.sprite.trigger_anim("back")
```

Comment:
Since this is the only place that uses back_sprite, the variable declaration could probably be removed from line 8.

Reply:
Actually, no. It has occurred to me that there's a reason this is here. If you talk to a toad, the same code is used, but you're not supposed to use the back sprite there.

Reply:
That should be fixed now.